examples/03-cpgf.py

```diff
@@ -9,7 +9,7 @@
 from pyscf import gto, scf, agf2, lib
 from dyson import CPGF, util
 
-ncheb = 50  # Number of Chebyshev moments
+ncheb = 100  # Number of Chebyshev moments
```

Comment:
was this a mistake? not that it really matters but which value is best for the example?

Reply:
This is a mistake. I didn't realize I committed the change.